### ISSUE TITLE
Migrate user subscription to learning path from c_item_property to a c_lp_rel_user table

### ISSUE THREAD

**Issue by ywarnier** [chamilo/chamilo-lms#3411]
The implementation of the subscription of users to a learning path has been made a bit quickly in 1.*, so the state of subscription is stored in c_item_property.lastedit_type = 'usersubscribe'.
Now that we're changing the c_item_property model, make sure this type or records is stored in a c_lp_rel_user table.
Something like
```
CREATE TABLE c_lp_rel_user (
    iid int primary key auto_increment,
    lp_id int,
    user_id int,
    c_id int,
    session_id int,
    created_at datetime,
    creator_id int
);
```

**Comment by christianbeeznest** [comment]
It is implemented in this PR https://github.com/chamilo/chamilo-lms/pull/4333 , checking the migration from 1.11 to 2.0

**Comment by NicoDucou** [comment]
This issue is solved with PR #4747
